docs/pages/desktop-access/active-directory.mdx

```diff
@@ -0,0 +1,213 @@
+---
```

Comment:
I can't figure out why GitHub shows this as a new and not renamed file, when the [commit](https://github.com/gravitational/teleport/pull/19311/commits/9a983a3592a1dbb452d02689ea28b0096509c015) shows it as a rename.